php/libraries/NDB_Menu_Filter.class.inc

```diff
@@ -668,6 +671,7 @@ class NDB_Menu_Filter extends NDB_Menu
         $x = 0;
         foreach ($this->list as $item) {
             //count column
+            // FIXME: Someone should investigate if this can use setTemplateVar
             $this->tpl_data['items'][$x][0]['value'] = $x + $count;
```

Comment:
There seems to be no good way to pass a sting to index a multidimensional array. Here's a stackoverflow thread on this topic https://stackoverflow.com/questions/7003559/use-strings-to-access-potentially-large-multidimensional-arrays